common/djangoapps/util/password_policy_validators.py

```diff
@@ -42,12 +44,12 @@ def validate_password_length(value):
     message = _("Invalid Length ({0})")
     code = "length"
 
-    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
-    max_length = getattr(settings, 'PASSWORD_MAX_LENGTH', None)
+    min_length = PASSWORD_MIN_LENGTH
```

Comment:
I believe you need to specify those settings on the edx/configuration repo. see https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml
You should consult with devops for sure